slackbridge/bots.py

```diff
@@ -71,8 +70,12 @@ def resolve(self):
                 channel_name = IRCBot.channels[channel_id]['name']
                 if message_type == 'message':
                     if 'subtype' in self.raw_message:
-                        if self.raw_message['subtype'] in IGNORED_MSG_SUBTYPES:
+                        subtype = self.raw_message['subtype']
+                        if subtype in IGNORED_MSG_SUBTYPES:
```

Comment:
Hmm. I wonder if we should do this check *before* we put messages in the queue.

Reply:
In either case, if we wanted to add handling for another type, we'd have to modify the list and add a handler. So it's largely a question of organization.

Reply:
Yeah, we could just not add it to the queue, but we have other message types we skip in the same way after we add them, so I think if we were going the route of not adding them to the queue we should do it to those messages too.